### ISSUE TITLE
Question about Yolo_loss class in train.py (output)

### ISSUE THREAD

**Issue by k223kim** [Tianxiaomo/pytorch-YOLOv4#119]
Hi, thank you for the awesome work!

I just have a question about the Yolo_loss_class in train.py.

In def forward in Yolo_class
`        for output_id, output in enumerate(xin):
            batchsize = output.shape[0]
            fsize = output.shape[2]
            n_ch = 5 + self.n_classes
            output = output.view(batchsize, self.n_anchors, n_ch, fsize, fsize)
            output = output.permute(0, 1, 3, 4, 2)  # .contiguous()`

I realized that xin has 3 outputs which each of the output has a bbox and its confidence. 

My question is, why are there 3 outputs in xin? Is that because we are using 3 yolo layers?

Also, if I want to calculate the precision of a particular batch when training, does that mean that I have to calculate the precision for each of the layer output? Or should I just use the precision of the very last layer?

Any help would be appreciated!

**Comment by Tianxiaomo** [comment]
Yes, because yolo has three outputs.
If you want to calculate accuracy, you need to combine the three outputs
